packages/feedback/src/screenshot/components/ScreenshotEditor.tsx

```diff
@@ -46,15 +46,17 @@ const constructRect = (box: Box): Rect => {
 };
 
 const getContainedSize = (img: HTMLCanvasElement): Box => {
+  const imgHeight = img.clientHeight;
```

Comment:
To not confuse it with `img.width` that's used below

```suggestion
  const imgClientHeight = img.clientHeight;
```